paywall/src/utils/withConfig.js

```diff
@@ -22,10 +22,11 @@ export default function withConfig(Component) {
     )
   }
 
-  function mapStateToProps(state) {
+  // We do not overwite existing props if any!
+  function mapStateToProps(state, props) {
     return {
-      network: state.network,
-      account: state.account,
+      network: props.network || state.network,
```

Comment:
This was causing the stories to fail because we had no account set in state.

Reply:
The "Insufficient story" was actually not showing insufficient funds :/

Reply:
is this the best option? Why not just set them in the state inside the stories? I ask because this increases surface area to look for bugs

Reply:
This was _actually_ a bug ;)